test/native/jest.config.js

```diff
@@ -55,6 +55,7 @@ module.exports = {
 		[ `@wordpress\\/(${ transpiledPackageNames.join(
 			'|'
 		) })$` ]: '<rootDir>/packages/$1/src',
+		'test/helpers$': '<rootDir>/test/native/helpers.js',
```

Comment:
This was required to resolve the path `test/helpers` for importing the helpers, I added this to prevent having a relative path too long.

Reply:
My personal thought is that we should avoid adding too many maps here, as it can become confusing as to where files originate with the mapping "magic" hidden. But I agree that something as ubiquitous as the testing helper is likely a good candidate. 👍🏻